### ISSUE TITLE
SRS final check

### ISSUE THREAD

**Issue by Pungent-Paws** [shaw-rohit/Hangman#33]
1. check SRS against Requirements Verification Checklist (the one shown during Blackbox lecture)
2. double check that code matches SRS (or the other way round ;) )

**Comment by arumoy-shome** [comment]
>more generally I wonder if it was indeed said that we don't need to exactly follow the SRS...

Yup we don't need to follow it to the dot, it's simply for reference.
